src/ui/screens/AccountsScreen.tsx

```diff
@@ -56,7 +56,8 @@ export function AccountsScreen({navigation}: Props) {
     navigation.setOptions({
       headerRight: HeaderRight,
     });
-  }, [navigation, HeaderRight]);
+    closeExternalAccount();
```

Comment:
Please, double check that there are not other issues introduced with this. Also, can you create an issue and add a TODO here with the link to keep track of it (If the library fixes the bug this should be removed)

Reply:
Ok, I'll verify that and add a TODO in the next commit.